<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firs Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="l_full_name">
       <property name="text">
        <string>Full name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_full_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="l_occupation">
       <property name="text">
        <string>Occupation :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_occupation"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pb_submit">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
